cv_next/app/feed/page.tsx

```diff
@@ -1,15 +1,11 @@
-import { Suspense } from "react";
-import { fetchCvs } from "@/app/actions/fetchCvs";
-import Feed from "./components/feed";
+import { Suspense } from "react"
+import Feed from "./components/feed"
 
 export default async function Page() {
```

Comment:
We don't want to initial-fetch on server?

Reply:
The fetching done at the feed itself(not the CV itemRSC) only gets the CV entry(the link, ID, author, etc) while the preview is at thee RSC

Reply:
In feed.tsx we do call the action fetchCvsForFeed which uses the getPaginatedCvs from the server

Reply:
Tagging here just to see if a follow-up is needed @YechiamW3